README.md

```diff
@@ -83,19 +83,28 @@ and starts listening on port 53 (override with `-dns.port`), it should show the
 
 ~~~ txt
 .:53
-CoreDNS-1.6.6
+CoreDNS-1.6.9
 linux/amd64, go1.13.5, aa8c32
 ~~~
 
+To query port 53 use the following:
+
+~~~ txt
+dig @localhost -p 53 www.example.com 
+~~~
+
 Any query sent to port 53 should return some information; your sending address, port and protocol
 used. The query should also be logged to standard output.
 
-If you have a Corefile without a port number specified it will, by default, use port 53, but you can
-override the port with the `-dns.port` flag: `coredns -dns.port 1053`, runs the server on port 1053.
+To modify the properties of the CoreDNS server, you will have to edit a file known as the `Corefile`. 
+If there is no `Corefile` in your coredns directory create one. Please do not get 
+confused between the Corefile and corefile.5.md (which is a file that contains more information about how
+to modify the Corefile.)
```

Comment:
Change above paragraph to the following:

```
The configuration of CoreDNS is done through a file named `Corefile`. When CoreDNS starts, it will look for the `Corefile` from the current working directory. A `Corefile` for CoreDNS server that listens on port `53` and enables `whoami` plugin is:

~~~ corefile
.:53 {
    whoami
}
~~~
```